google_guest_agent/oslogin.go

```diff
@@ -121,6 +130,8 @@ func (o *osloginMgr) Set(ctx context.Context) error {
 	oldEnable, _, _ := getOSLoginEnabled(oldMetadata)
 	enable, twofactor, skey := getOSLoginEnabled(newMetadata)
 
+	cleanupDeprecatedDirectives()
```

Comment:
should this be skipped if agent has updated once before (keeping a cache of file update)? Otherwise on every `Set()` agent will try finding diff and overwriting file with no change

Reply:
How about "only overwrite the content" if it changed, meaning, we couldn't find any deprecated content?

Reply:
done. thanks.